src/presentation/components/organisms/DoubleUserDataPanel.tsx

```
@@ -1,54 +1,86 @@
+            obj[header] = values[idx] ?? "";
+          });
+          return obj;
+        });
+        return { headerList, parsedRows };
+      };
+
+      const first = parseTable(sections[0]);
+      const second = parseTable(sections[1]);
+
+      setHeaders(first.headerList);
+      setFilters(first.headerList.reduce((acc, key) => ({ ...acc, [key]: true }), {}));
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)